python/common/org/python/stdlib/datetime/TimeDelta.java

```diff
@@ -192,12 +296,134 @@ public TimeDelta(org.python.Object[] args, java.util.Map<java.lang.String, org.p
         String days = Long.toString(dayslong);
         long seconds = ((org.python.types.Int) this.seconds).value;
         long microseconds = ((org.python.types.Int) this.microseconds).value;
-        String returnStr = days + " days, " + "seconds: " + seconds + ", microseconds: " + microseconds;
+        String returnStr = "days: " + days + ", seconds: " + seconds + ", microseconds: " + microseconds;
```

Comment:
This format does not seem correct. :thinking: 
 For example does `timedelta(1,2,3).__str__()` return `'1 day, 0:00:02.000003'
`

Reply:
Good catch! Fixed in 6296ecf401e592553c038b6f16ab6e1e2299af73